tests/test_cli.py

```diff
@@ -102,5 +113,22 @@ def test_database_for_unsupported_song(self):
 		"""
 		self.assertEqual(get_lyrics("Bitch Lasagna", "Party in Backyard")[:7], "[Intro]")
 
+	@mock.patch('requests.post', return_value=R())
+	def test_that_get_lyrics_does_not_break_with_request_giving_wrong_status_code(self, mock_requests):
+		"""
+		Test the get_lyrics does not break with requests giving wrong status code
+		"""
+		self.assertEqual(get_lyrics("Ki", "Ki", True), "Couldn\'t get lyrics for Ki by Ki.\n")
+
+	@mock.patch('requests.post', side_effect=requests.exceptions.RequestException)
```

Comment:
can't this and the above test be combined?

Reply:
I don't see the point. Both tests check different parts of code